application/templates/_shared/_header.html

```diff
@@ -10,7 +10,7 @@
     <div class="header-proposition">
       <div class="content">
         <nav id="proposition-menu">
-          <span itemprop="name"><a itemprop="url" href="{{ config.RDU_SITE | strip_trailing_slash }}{{ url_for('static_site.index') }}" id="proposition-name">
+          <span itemprop="name"><a itemprop="url" href="{{ (config.RDU_SITE | strip_trailing_slash) if static_mode else "" }}{{ url_for('static_site.index') }}" id="proposition-name">
```

Comment:
@samuelhwilliams could we switch this back to using `_external=True` by renaming `RDU_SITE` to `SERVER_NAME`? See http://flask.pocoo.org/docs/1.0/api/#flask.url_for

Reply:
`SERVER_NAME` has some caveats that I'm not happy assessing for this bugfix. But we can look into it as a separate thing.

Reply:
@samuelhwilliams ok. I think though we can actually remove the whole of `{{ config.RDU_SITE | strip_trailing_slash }}` as I _think_ Google can resolve the full URL from the relative one when actually crawling the site (but just can't when you're using the testing tool and pasting in the HTML source).

I'm slightly loathe to add any more `if static_mode` conditions to the templates, in order to make it easier for us to switch to using the Flask site as the origin server for our CDN instead of the static build at some future point...

Reply:
`if static_mode` is an easy thing to search for in templates and is probably the least of our worries if/when we come around to doing that.

Reply:
Okay fine. 😄